client/client-multi/client-cpp/Modules/FindSqlite3.cmake

```diff
@@ -37,31 +29,8 @@ IF (APPLE)
   ENDIF ()
 ENDIF (APPLE)
 
-IF ($ENV{SQLITE_ROOT})
```

Comment:
I don't think this "feature" was documented at all, so the impact on the end user is minimal.